contrib/namespace_cleanup.sh

```diff
@@ -5,14 +5,20 @@ ecosystem="stagef"
 read -p "Are you at tron-$ecosystem (y/n)?" RES
 echo
 if [ $RES = "y" ]; then
-        for file in /nail/tron/config/*
+        #load namespace from _manifest.yaml
+        for namespace in $(cat /nail/tron/config/_manifest.yaml | uq | jq -r 'keys[]')
         do
-                filename="$(basename $file)"
-                namespace="$(basename $file .yaml)"
+                file=$(cat /nail/tron/config/_manifest.yaml | uq | jq -r .\"$namespace\")
+                filename=$(basename $file)
                 if [ -f "/nail/etc/services/tron/$ecosystem/$filename" ]; then
                         echo "$namespace is up to date"
+                elif [ $namespace == "MASTER" ]; then
+                        echo "It is MASTER namepsace"
```

Comment:
This means that MASTER doesn't exist in /nail/etc/services/tron/$ecosystem/, right? If we're going to cover that case, we should make the message seem more dire :)

Reply:
No, it is just because of MASTER namespace mapping to master.yaml in _manifest.yaml. We could check if MASTER.yaml exists. I think the scenario with no MASTER.yaml is pretty rare, and if it does happen, we might know that before this script running.